tests/dialects/test_sqlite.py

```diff
@@ -140,6 +140,19 @@ def test_sqlite(self):
             read={"snowflake": "LEAST(x, y, z)"},
             write={"snowflake": "LEAST(x, y, z)"},
         )
+        self.validate_all(
+            "SELECT STR_TO_DATE('2022-01-01', '%Y-%m-%d')",
+            write = {
+                        "mysql": "SELECT STR_TO_DATE('2022-01-01', '%Y-%m-%d')",
+                        "sqlite": "SELECT STRFTIME('%Y-%m-%d', '2022-01-01')"
+                    }
+        )
```

Comment:
Run `make style` for commits that are part of a PR so that the CI doesn't crash due to styling / linter issues